core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -5208,6 +5203,79 @@ components:
           maxLength: 64
           description: |
             The hex-encoded root hash of the receipt tree. This captures the consensus-agreed output of each transaction on the ledger.
+##############################
+# REQUEST: /status/scenarios #
+##############################
+    ScenariosRequest:
+      type: object
+      required:
+        - network
+      properties:
+        network:
+          $ref: "#/components/schemas/NetworkIdentifier"
+    ScenariosResponse:
+      type: object
+      required:
+        - executed_scenarios
+      properties:
+        executed_scenarios:
+          type: array
+          items:
+            $ref: "#/components/schemas/ExecutedGenesisScenario"
+          description: Scenarios executed as part of Genesis (in their execution order).
+    ExecutedGenesisScenario:
+      type: object
+      required:
+        - sequence_number
+        - logical_name
+        - committed_transactions
+        - addresses
+      properties:
+        sequence_number:
+          type: integer
+          format: int32
+          minimum: 0
+          maximum: 1000000
+          description: |
+            An index of the Scenario on the list of all Scenarios that were executed.
+            Note: the stored sequence numbers do not necessarily have to be consecutive (e.g. in a
+            case where some configured Scenario failed to execute or failed to write results to the
+            database).
+        logical_name:
+          type: string
+        committed_transactions:
+          type: array
+          items:
+            $ref: "#/components/schemas/ExecutedScenarioTransaction"
+          description: Transactions successfully committed by the Scenario.
+        addresses:
+          type: array
+          items:
+            $ref: "#/components/schemas/DescribedAddress"
+          description: Well-named addressed touched/created by the Scenario.
+    ExecutedScenarioTransaction:
+      type: object
+      required:
+        - logical_name
+        - state_version
+        - intent_hash
+      properties:
+        logical_name:
+          type: string
+        state_version:
+          $ref: "#/components/schemas/StateVersion"
+        intent_hash:
+          $ref: "#/components/schemas/IntentHash"
+    DescribedAddress:
+      type: object
+      required:
+        - logical_name
+        - address
+      properties:
+        logical_name:
+          type: string
+        address:
+          $ref: '#/components/schemas/GlobalAddress'
```

Comment:
Just a heads up that in the next PR, these will become more general addresses - ie include Vault addresses and even non fungible global ids - so perhaps we should just make this a string with a description.

Reply:
Alright, I migrated to a string with more vague description here 👍